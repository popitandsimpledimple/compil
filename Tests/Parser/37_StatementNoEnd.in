var a : integer;
begin
a := 1;
    begin;
    
end.