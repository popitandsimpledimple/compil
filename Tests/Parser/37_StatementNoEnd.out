(6,4) Expected 'Semicolom' Found 'Dot'